src/structures/ApplicationCommand.js

```diff
@@ -136,6 +136,10 @@ class ApplicationCommand extends Base {
 
   /**
    * An option for an application command or subcommand.
+   * <info>In addition to the listed properties, when used as a parameter,
+   * API style `snake_case` properties can be used for compatibility with generators like `@discordjs/builders`.</info>
```

Comment:
Since `@discordjs/builers` has documentation now, maybe we can link it?

Reply:
Separate PR, if needed.